docs/sources/variables/advanced-variable-format-options.md

```diff
@@ -148,4 +148,22 @@ Formats single- and multi-valued variables into their text representation. For a
 servers = ["test1", "test2"]
 String to interpolate: '${servers:text}'
 Interpolation result: "test1 + test2"
-```
\ No newline at end of file
+```
+
+## Replace
+
+Replaces text with single and multi-valued variables. It accepts one to three arguments and allows the escaping of colons using backslashes. When called with two or more arguments, the first two arguments follow the semantics of `String.prototype.replace(. ) `. When called with an odd number of arguments, the rightmost argument should be either `upper` or `lower`, which will change the case of the result string.
+
+```bash
+servers = ["value1", "value2"]
+String to interpolate: '${servers:replace:upper}'
+Interpolation result: "VALUE1,VALUE2"
+
+servers = ["value1", "values2"]
+String to interpolate: '${servers:replace:values?:val}'
+Interpolation result: "val1,val2"
+
+servers = ["value1", "values2"]
+String to interpolate: '${servers:replace:values?:val:upper}'
```

Comment:
I just run upon this PR coming from an Issue. If my understanding is correct, we should change the documentation as follows: 🙏 

```suggestion
servers = ["value1", "value2"]
String to interpolate: '${servers:replace:value?:val}'
Interpolation result: "val1,val2"

servers = ["value1", "value2"]
String to interpolate: '${servers:replace:value?:val:upper}'
```